package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webnative",
-  "version": "0.36.3",
+  "version": "0.37.0",
```

Comment:
For sure. What's the convention here? Just `next` or something like `next-unix_timestamp_goes_here`?

Reply:
I've seen a couple of approaches to using `next`.

SvelteKit used `next` in the version while they were working towards a `1.0.0` release: https://www.npmjs.com/package/@sveltejs/kit?activeTab=versions. For example, you could install `1.0.0-next.567`. They also tagged each next release so you could install `@next` to get the most recent next.

`npm` does next versions using only tags: https://www.npmjs.com/package/npm?activeTab=versions. It looks like they run next alongside stable versions until they reach the next major release.

Neither of these options seem to be exactly what we want because they both work towards a major release.

Reply:
Nice! 👀 Updated it to `next`